### DISCUSSION TITLE
When importing files are they copied to Wallpaper Directory

### DISCUSSION THREAD

**Discussion by iSupremacy** [rocksdanister/lively#2196] [Q&A]
im just curious when we import files, pictures/videos etc are files being copied into that "Wallpaper Directory"? Because there's that little progress bar when you import so I kinda got that impression. I just want to know so I can redirect that off the Cdrive if necessary.
cheers!

**Comment by rocksdanister** [comment]
No, only Lively .zip files (... -> Save to file.) are installed in Wallpaper Directory, rest only the location is saved.

The progress is for the animated preview gif creation.
